Queueomatic/Server/Endpoints/Participant/Delete/DeleteParticipantValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FastEndpoints;
+using FluentValidation;
+
+namespace Queueomatic.Server.Endpoints.Participant.Delete;
+
+public class DeleteParticipantValidator : Validator<DeleteParticipantRequest>
+{
+    public DeleteParticipantValidator()
+    {
+        RuleFor(expression => expression.Id)
```

Comment:
Maybe check if you can parse it to a guid? It is possible to do it with a custom validator. For example with ["must"](https://docs.fluentvalidation.net/en/latest/custom-validators.html).

Reply:
It checks by default because the request type is Guid.